includes/models/model.llms.course.php

```diff
@@ -548,6 +548,15 @@ public function has_prerequisite( $type = 'any' ) {
 
 	}
 
+	/**
+	 * Whether the course lessons must be completed sequentially
+	 *
+	 * @return bool
+	 */
+	public function must_complete_sequentially(){
```

Comment:
Need to add tests for this method in the course class

Reply:
Done